src/main/resources/sun_checks_modified.xml

```diff
@@ -107,10 +107,10 @@
 
         <!-- Checks for imports                              -->
         <!-- See https://checkstyle.org/checks/imports/index.html -->
-        <module name="AvoidStarImport"/>
+<!--        <module name="AvoidStarImport"/>-->
```

Comment:
Approved but curious why the deviation from the style guidance https://checkstyle.org/checks/imports/avoidstarimport.html#AvoidStarImport

Reply:
Just for today's task we are being less strict on linting, and for some reason when I used intelliJ's import optimise feature it bundled some imports into a .* type